Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -293,6 +292,9 @@ function CesiumWidget(container, options) {
     if (!defined(globe)) {
       globe = new Globe(ellipsoid);
     }
+    if (defined(options.terrainExaggeration)) {
+      globe.terrainExaggeration = options.terrainExaggeration;
+    }
```

Comment:
Should this have the deprecation warning too?

Reply:
Now when you do
```
var viewer = new Cesium.Viewer("cesiumContainer", {
  terrainProvider: Cesium.createWorldTerrain(),
  terrainExaggeration: 2.0
});
```
it will say: `terrainExaggeration is now a property of Globe`